### DISCUSSION TITLE
What is the way to define the target directory of Generate REST API

### DISCUSSION THREAD

**Discussion by hguerrero** [apache/camel-karavan#794] [Q&A]
When managing an open api in one of the subfolders, I always get the yaml integration generated in the parent folder. 

Is there a way to define the target directory for the newly created integration?

**Comment by davsclaus** [comment]
its jbang that generates

camel generate rest --help

There is an -o for output

When you say "managing in sub folders". Can you maybe detail your folder structure for us to better understand. And I assume you use Karavan UI to generate.

**Reply by hguerrero** [reply]
This is a similar structure:

```
projects/
├─ openapi-project/
│  ├─ openapi-spec.json
│  ├─ route.camel.yaml <--- Correct
route.camel.yaml <--- Wrong!
```

When selecting the `openapi-spec.json` file, I expected the call to *Karavan: Generate REST API* to create the corresponding Camel route in the same folder as the openapi. However, the file is created in the root folder of the code, under `/projects`.
